### PR TITLE
Add Book Binder

### PR DESCRIPTION
**Pull request by svr333**
## Changes
- Adds the Book Binder page to the Electric Machines category

### CODE REVIEW

**Comment by WalshyDev** [comment]
We should really go into config options for these things too.

**Comment by svr333** [comment]
sure

**Comment by svr333** [comment]
Something like this you had in mind @WalshyDev?

**Comment by WalshyDev** [comment]
I meant going into what the specific options are. End of the day, this is a wiki. It should have as much info as it can

**Comment by svr333** [comment]
I wasnt sure whether to keep the little text on top of the table

Soon Ill also make a pr to add this for all existing machines

File: pages/Book-Binder.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+| Name                     | Type    | Default Value |
+| ------------------------ | ------- | ------------- |
+| bypass-vanilla-max-level | boolean | false         |
+| has-custom-max-level     | boolean | false         |
+| custom-max-level         | int     | 15            |
+
+**Tip:** Updating machine specific settings can be done in items.yml
+
+## Book Binder Tiers
```
Comment (WalshyDev) [inline pages/Book-Binder.md:15]: I'd probably have tiers above settings. That's what players will care about and players outnumber server owners

File: pages/Book-Binder.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+The Book Binder is an [Electric Machine](https://github.com/Slimefun/Slimefun4/wiki/Electric-Machines) which can be used to combine different books together (much like an anvil would).  
+
+## Book Binder Settings
+By default, you cannot use this machine to get books above the vanilla limit.  
+However for server owners it is possible to adjust this and either add a custom limit, or no limit at all.
+
+| Name                     | Type    | Default Value |
+| ------------------------ | ------- | ------------- |
+| bypass-vanilla-max-level | boolean | false         |
```
Comment (WalshyDev) [inline pages/Book-Binder.md:9]: most settings names are pretty descriptive but think we need a description too?

**Review by WalshyDev — CHANGES_REQUESTED** [review]

File: pages/Book-Binder.md (hunk 2 above)
Comment (svr333) [reply pages/Book-Binder.md:9]: uh sure Ill just use a summary so it doesnt get too big

File: pages/Book-Binder.md (hunk 1 above)
Comment (svr333) [reply pages/Book-Binder.md:15]: fair point

**Comment by WalshyDev** [comment]
I don't the summaries are needed

**Review by WalshyDev — APPROVED** [review]
LGTM

**Pull request merged**